packages/frontend/src/components/SettingsValidator/ValidatorTable/index.tsx

```diff
@@ -107,63 +106,47 @@ const TableValidationItem = forwardRef<
             }
         >
             <td>
-                <Anchor
-                    sx={{
-                        color: 'unset',
-                        ':hover': {
-                            color: 'unset',
-                            textDecoration: 'none',
-                        },
-                    }}
-                    href={getLinkToResource(validationError, projectUuid)}
-                    target="_blank"
```

Comment:
I see the benefit in keeping this `Anchor`, so users can copy the row link.  Could we instead do something with the click event in `handleOnValidationErrorClick`? 

Or remove the click event and keep the `Anchor`?

Reply:
@joaoviana updated PR and preserved old behavior. its not possible to semantically wrap table with anchor without doing some hacks so this should be good enough 👍